### DISCUSSION TITLE
Support for MT5!

### DISCUSSION THREAD

**Discussion by ricardodacosta** [SmartLever/SmartBots#2] [Ideas]
I would like to request you add support for MT5. This would allow US customers to use your service to trade Futures market with AMP FUTURES. They do not offer MT4.

Also a question...does it work with any broker that supports MT4 ? For example OANDA in the US offers MT4 for FX trading 

Thank you!

**Comment by SmartLever** [comment]
Hi, Thank you very much for the request, in future updates we want to implement MT5, and regarding the other question, SmartBots works for any broker that supports mt4, so you can use OANDA for FX trading, if you have more questions or want something more specific, I'll leave you my email: andres@smartlever.tech
